pyrsp/utils.py

```diff
@@ -16,10 +16,11 @@
 
 # (C) 2014 by Stefan Marsiske, <s@ctrlc.hu>
 
-from socket import SO_REUSEADDR, SOL_SOCKET, socket, AF_INET, SOCK_STREAM
+from socket import socket, AF_INET, SOCK_STREAM
 from time import time
 from json import JSONDecoder, JSONEncoder
 from six import PY3
+from psutil import net_connections
```

Comment:
is psutil a new dependency or is psutil packaged in the core py modules?

Reply:
`psutil` is  a new dependency.

Reply:
ah, then we should add it to setup.py also i guess.

Reply:
actually i saw the import. i was just not sure psutil was not already used elsewhere